app/components/homeScreen/DailyReads.tsx

```diff
@@ -164,18 +169,18 @@ const DailyReads = (): any => {
       if (dailyDataCategoryall[activeChild.uuid].taxonomyid != activeChild.taxonomyData.id) {
         advicearr = [];
       }
-      if (dailyDataCategoryall[activeChild.uuid].prematureTaxonomyId != activityTaxonomyId) {
+      if (dailyDataCategoryall[activeChild.uuid]?.prematureTaxonomyId != activityTaxonomyId) {
         gamesarr = [];
       }
       showedDailyDataCategory = { advice: advicearr, games: gamesarr };
     }
     const nowDate = DateTime.now().toISODate();
     if (dailyDataCategory && (dailyDataCategory.currentDate == '' || dailyDataCategory.currentDate < nowDate)) {
       let filteredArticles;
-      if (activeChild.isPremature === 'true') {
-        filteredArticles = ArticlesData.filter((article: any) => article.premature === 1).sort((a: any, b: any) => new Date(b.created_at) - new Date(a.created_at));
-        ArticlesData = filteredArticles;
-      }
+      // if (activeChild.isPremature === 'true') {
+      //   filteredArticles = ArticlesData.filter((article: any) => article.premature === 1).sort((a: any, b: any) => new Date(b.created_at) - new Date(a.created_at));
+      //   ArticlesData = filteredArticles;
+      // }

```

Comment:
We should remove the commented code if it is not needed anymore.

Reply:
we temporary commented this code we need in future 
i added proper comment for this